signale.d.ts

```diff
@@ -0,0 +1,148 @@
+// Type definitions for signale
+// Project: https://github.com/klauscfhq/signale
+// Definitions by: Resi Respati <https://github.com/resir014>, Kingdaro <https://github.com/kingdaro>, Joydip Roy <https://github.com/rjoydip>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.7
+
+/// <reference types="node" />
+
+interface Map<v, k> {}
+interface Set<T> {}
```

Comment:
These interfaces should be included in the TS stdlib, so they can be removed.

Reply:
TS lib not including those. I have tried a couple of solutions and lastly, I got this solution [here](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/8264#issuecomment-189247585).

Reply:
I think we can still bump `"lib"` up to `"es6"` inside the tsconfig. That way both `Map` and `Set` are included in the stdlib, and these two lines could be removed.

Besides, the library is already based on es6 classes anyway :)

Reply:
I had tried too.

Reply:
![capture](https://user-images.githubusercontent.com/15318294/41202917-fdfa3a36-6cef-11e8-929d-35a4de9a5a65.PNG)

Reply:
Actually, disregard my last comment. I just cloned your branch locally and just saw what you mean. It's safe to still keep these two declaration there.

Ignore my last comment, thanks. :)